src/schema/system_time.js

```diff
@@ -0,0 +1,39 @@
+import gravity from "lib/apis/gravity" // Uncached
+import {
+  GraphQLString,
+  GraphQLBoolean,
+  GraphQLInt,
+  GraphQLObjectType,
+} from "graphql"
+
+const SystemTimeType = new GraphQLObjectType({
+  name: "SystemTime",
+  fields: () => {
+    return {
+      day: { type: GraphQLInt },
+      wday: { type: GraphQLInt },
+      month: { type: GraphQLInt },
+      year: { type: GraphQLInt },
+      hour: { type: GraphQLInt },
+      min: { type: GraphQLInt },
+      sec: { type: GraphQLInt },
+      dst: { type: GraphQLBoolean },
+      unix: { type: GraphQLInt },
+      utc_offset: { type: GraphQLInt },
+      zone: { type: GraphQLString },
+      iso8601: { type: GraphQLString },
+    }
+  },
+})
+
+const SystemTime = {
+  type: SystemTimeType,
+  description:
+    "Gravity system time, necessary for synchronizing device clocks.",
+  resolve: async () => {
+    const { body } = await gravity("system/time")
```

Comment:
I think this should still use a data loader, vs the more direct gravity fetch.

Reply:
Given that it's a non-trivial (meaning, I don't think we have other implementations yet in MP) use of a data loader because it will be unauthorized but also must not be cached, can we merge this now and follow up with that optimization? This should be sufficient for the current use-case and will unblock our emission work!

Reply:
Sure, it may not be 'trivial', but its pretty standard I think, just `new DataLoader(Promise, { cache: false })` basically.

You can probably use the 'one off' factory for this- https://github.com/artsy/metaphysics/blob/c43bb86bc7354d1f42fc162491b20995b3861ba7/src/lib/loaders/api/loader_one_off_factory.js, looks like thats exactly what it does.

Reply:
Oh! That's cool. Thought it would need a whole new factory and such. I'll make this change 👍